src/Geocodio.php

```diff
@@ -37,7 +39,25 @@ class Geocodio
         'country',
     ];
 
-    public function __construct(private readonly Client $client = new Client) {}
+    const SDK_VERSION = '1.2.0';
+
+    public function __construct(private readonly Client $client = new Client)
+    {
+        $this->apiKey = getenv('GEOCODIO_API_KEY');
+
+        if ($hostname = getenv('GEOCODIO_HOSTNAME')) {
+            $this->hostname = $hostname;
+        }
+
+        if ($apiVersion = getenv('GEOCODIO_API_VERSION')) {
+            $this->apiVersion = $apiVersion;
+        }
+    }
```

Comment:
Not sold on this but I kind of like the idea of defaulting to the ENV vars. I think I might refactor this to constructor ags (updating the service provider) and setting up client overrides via `->withClient(Client $client)`